src/components/pages/ItemResults.js

```diff
@@ -19,38 +21,48 @@ const useStyles = makeStyles((theme) => ({
     flexDirection: "column",
     alignItems: "center",
   },
-}));
+});
 
 // this is the page with all the item previews displayed (also includes the result to searching for a specific item)
-const ItemResults = ({ postings }) => {
-  const classes = useStyles();
-  return (
-    <Container className={classes.root}>
-      <div className={classes.search}>
-        <SearchBar />
-      </div>
-      <Grid container direction={"row"} spacing={4}>
-        {postings.map((item, index) => {
-          return (
-            <Grid key={index} item xs={3}>
-              <ItemPreview
-                id={item.id}
-                title={item.title}
-                datePosted={item.datePosted}
-                location={item.location}
-                images={item.images}
-                postings={postings} //TODO: remove when BE implemented
-              />
-            </Grid>
-          );
-        })}
-      </Grid>
-    </Container>
-  );
-};
+class ItemResults extends React.Component {
+  componentDidMount() {
+    let val = this.props.location.search.replace("?", "");
+    this.props.loadPostingsByQuery(val);
+  }
+
+  render() {
+    const { classes, postings } = this.props;
+
+    return (
+      <Container className={classes.root}>
+        <div className={classes.search}>
+          <SearchBar />
+        </div>
+        <Grid container direction={"row"} spacing={4}>
+          {postings.map((item, index) => {
+            return (
+              <Grid key={index} item xs={3}>
+                <ItemPreview
+                  _id={item._id}
+                  title={item.title}
+                  date={item.date}
+                  location={item.location}
+                  images={item.images}
+                  postings={postings} //TODO: remove when BE implemented
```

Comment:
is this supposed to be removed now that BE is implemented?

Reply:
Ah yeah! I missed these. Will give it another pass to clear these out tomorrow. Sorry!